### PR TITLE
Feature/file desc visualization

### PR DESCRIPTION
**Pull request by ChiaraDipi**
Implementation of the component to visualize data retrieved from `<fileDesc>` element.
In order to test it, you can add
```
<evt-content-viewer [content]="testEl$ | async"></evt-content-viewer>
```
in `app.component.html` and 

```
public testEl$ = this.evtModelService.editionSource$.pipe(
  map(s => {
    const fileDesc = s.querySelector<HTMLElement>('fileDesc');
    const fileDescParser = createParser(FileDescParser, parse);

    return fileDescParser.parse(fileDesc);
  }),
  tap(e => console.log(e)),
);

constructor(
  private evtModelService: EVTModelService,
) {
}
```
in `app.component.ts` and of course have a complete `<fileDesc>` element in your TEI xml file.

### CODE REVIEW

File: src/app/components/extent/extent.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<evt-header-section *ngIf="data?.content?.length > 0" [label]="'extent'" [additionalClass]="'extent'" [inlineLabel]="false">
```
Comment (szenzaro) [inline src/app/components/extent/extent.component.html:1]: Shouldn't the label be translated as well?

File: src/app/components/publication-stmt/publication-stmt.component.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<h4 *ngIf="data?.content?.length > 0" class="main-section-title">{{ 'publicationStatement' | translate }}</h4>
+<ng-container [ngSwitch]="data?.structuredData">
+    <ng-container *ngSwitchCase="true">
+        <ng-container *ngTemplateOutlet="section; context: { label: 'publisher', items: data?.publisher, class: 'publisher' }"></ng-container>
```
Comment (szenzaro) [inline src/app/components/publication-stmt/publication-stmt.component.html:4]: Shouldn't the labels be translated?

File: src/app/components/series-stmt/series-stmt.component.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<h4 *ngIf="data?.content?.length > 0" class="main-section-title">{{ 'seriesStatement' | translate }}</h4>
+<div class="seriesStatement">
+    <ng-container [ngSwitch]="data?.structuredData">
+        <ng-container *ngSwitchCase="true">
+            <ng-container *ngTemplateOutlet="section; context: { label: '', items: data?.title, class: 'title' }"></ng-container>
+            <ng-container *ngTemplateOutlet="section; context: { label: '', items: data?.biblScope, class: 'biblScope' }"></ng-container>
+            <ng-container *ngTemplateOutlet="section; context: { label: '', items: data?.editor, class: 'editor' }"></ng-container>
+            <ng-container *ngTemplateOutlet="section; context: { label: 'idno', items: data?.idno, class: 'idno' }"></ng-container>
+            <ng-container *ngTemplateOutlet="section; context: { label: 'responsibles', items: data?.respStmt, class: 'respStmt', itemsSeparator: ', ' }"></ng-container>
```
Comment (szenzaro) [inline src/app/components/series-stmt/series-stmt.component.html:9]: Translate label

File: src/app/components/series-stmt/series-stmt.component.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        </ng-container>
+        <ng-container *ngSwitchDefault>
+            <evt-content-viewer *ngFor="let element of data?.content" [content]="element"></evt-content-viewer>
+        </ng-container>
+    </ng-container>
+</div>
+
+<ng-template #section let-label="label" let-items="items" let-class="class" let-itemsSeparator="itemsSeparator">
+    <evt-header-section *ngIf="items?.length > 0" [label]="label" [additionalClass]="class" [inlineLabel]="true">
```
Comment (szenzaro) [inline src/app/components/series-stmt/series-stmt.component.html:18]: translate label

File: src/app/components/title-stmt/title-stmt.component.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    <evt-content-viewer [content]="el"></evt-content-viewer>
+</div>
+<div class="sub-title" *ngFor="let el of data?.subtitles" [evtHtmlAttributes]="el?.attributes">
+    <evt-content-viewer [content]="el"></evt-content-viewer>
+</div>
+<div class="author" *ngFor="let author of data?.authors">
+    <evt-content-viewer [content]="author"></evt-content-viewer>
+</div>
+
+<ng-container *ngTemplateOutlet="section; context: { label: 'editors', items: data?.editors, class: 'editor' }"></ng-container>
```
Comment (szenzaro) [inline src/app/components/title-stmt/title-stmt.component.html:11]: labels need transaltion

File: src/app/components/edition-stmt/edition-stmt.component.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<h4 *ngIf="data?.content?.length > 0" class="main-section-title">{{ 'editionStatement' | translate }}</h4>
+<ng-container [ngSwitch]="data?.structuredData">
+    <ng-container *ngSwitchCase="true">
+        <ng-container *ngTemplateOutlet="section; context: { label: 'edition', items: data?.edition, class: 'edition' }"></ng-container>
```
Comment (szenzaro) [inline src/app/components/edition-stmt/edition-stmt.component.html:4]: Translate labels

**Review by szenzaro — CHANGES_REQUESTED** [review]

**Review by szenzaro — APPROVED** [review]

**Review by giuliac89 — APPROVED** [review]

**Pull request merged**